testing/trino-tests/src/test/java/io/trino/security/TestAccessControl.java

```diff
@@ -954,6 +958,34 @@ public void testPropertiesAccessControlIsSkippedWhenUsingDefaults()
         assertAccessAllowed("CREATE MATERIALIZED VIEW mock.default.new_materialized_view AS SELECT 1 a");
     }
 
+    @Test
+    public void testAccessControlWithGroupsAndColumnMask()
+    {
+        String securityConfigFile = getResource("access_control_rules_with_groups.json").getPath();
+        systemAccessControl.set(new FileBasedSystemAccessControl.Factory()
```

Comment:
Can we use something like `DenySetPropertiesSystemAccessControl` to mimic the behavior `FileBasedSystemAccessControl`? Then the existing tests can be moved to `BaseFileBasedConnectorAccessControlTest`. WDYT?